### DISCUSSION TITLE
webpage with video overlay on text

### DISCUSSION THREAD

**Discussion by carlo2222** [tailwindlabs/tailwindcss#12883] [Help] [answered]
when I open the page the video overlaps the text I can't find the error
![Screenshot (185)](https://github.com/tailwindlabs/tailwindcss/assets/129618933/9b871644-fd68-4618-99e1-45e3e481dd12)

**Comment by wongjn** [comment, marked as answer]
Please consider providing a [Tailwind Play](https://play.tailwindcss.com) or a project that reproduces the undesired behavior.

**Reply by carlo2222** [reply]
ok here is a codes part with the trouble https://play.tailwindcss.com/aUYUdIyAje.
Tt works on mobile wiev but not on PC
